### PR TITLE
Update package-spec (validate Kibana filenames)

### PR DESCRIPTION
**Pull request by mtojek**
This PR updates the dependency on package-spec to validate Kibana filenames.

### CODE REVIEW

File: test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json [hunk 1]
```diff
@@ -148,46 +148,46 @@
             }
         ],
         "timeRestore": false,
-        "title": "[Logs Apache] Access and error logs ECS",
```
Comment (ycombinator) [inline test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: Strictly speaking, is this change needed as a result of updating the `package-spec` dependency? IIRC we don't yet have validation for this in the package spec, right?

File: test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json [hunk 2]
```diff
@@ -148,46 +148,46 @@
...
+        "title": "[Logs Apache] Access and error logs",
         "version": 1
     },
-    "id": "apache-Logs-Apache-Dashboard-ecs",
```
Comment (ycombinator) [inline test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:154]: Same question as https://github.com/elastic/elastic-package/pull/154/files#r514378108.

File: test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json (hunk 1 above)
Comment (mtojek) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: I admit I blindly copied over files from Integrations, which have been adjusted. If you prefer I can prepare anothe PR for spec to prevent `-ecs` suffixes and then bump up dependencies. WDYT?

File: test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json (hunk 2 above)
Comment (mtojek) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:154]: Response above

File: test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json (hunk 1 above)
Comment (ycombinator) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: Yeah, I think I would prefer a separate PR. We're going to need it anyway to bump up dependencies when we implement validation for the absence of the `-ecs` prefix in the package spec (https://github.com/elastic/package-spec/issues/71). It would be nice if that PR also contained the corresponding changes to the test packages in `elastic-package` as well. That way, it's clearer what the change is about.

Comment (mtojek) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: I opened another PR with no-ECS-suffix validation: https://github.com/elastic/package-spec/pull/74

Comment (mtojek) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: The spec has been updated. Let me work on this PR to make it back to green.

Comment (mtojek) [reply test/packages/apache/kibana/dashboard/apache-Logs-Apache-Dashboard.json:151]: The PR is back to green and looks fine.

**Review by ycombinator — APPROVED** [review]
LGTM.
